src/08-accessing.js

```diff
@@ -103,7 +114,9 @@ function getFirstPhoneNumber(person) {}
  *  getLastPhoneNumber(person);
  *  //> 7185558611
  */
-function getLastPhoneNumber(person) {}
+function getLastPhoneNumber(person) {
+  return person.numbers.pop()
+}
```

Comment:
This is another method that changes the array (and the object it's on). This is the source of your bug, I believe!